app/services/getauthorbyisbn_service.rb

```diff
@@ -0,0 +1,22 @@
+class GetauthorbyisbnService < ApplicationService
```

Comment:
This is good @khitishpradhan. Nice work ! 👍

Reply:
thanks @RathanKumar